pkg/bbgo/order_executor_general.go

```diff
@@ -162,6 +163,27 @@ func (e *GeneralOrderExecutor) BindProfitStats(profitStats *types.ProfitStats) {
 	})
 }
 
+func (e *GeneralOrderExecutor) BindProfitTracker(profitTracker *report.ProfitTracker) {
+	e.session.Subscribe(types.KLineChannel, profitTracker.Market.Symbol, types.SubscribeOptions{Interval: profitTracker.Interval})
```

Comment:
note the "Subscribe" call must be done before we connect the websocket server

Reply:
Do you mean put those subscribe call in Strategy.Subscribe() ? Is it executed before connecting to the websocket server?

Reply:
yes

Reply:
done